js/src/ext/dropdown.js

```diff
@@ -0,0 +1,53 @@
+Wu.Dropdown = Wu.Class.extend({
+    // this function will run automatically on new Wu.Dropdown()
+	initialize : function (options) {
+    	var me = this;
```

Comment:
please don't use `me` for context. `this` is used everywhere else, so we should stick to `this`.